test/portworx/storagecluster_helm_template_test.go

```diff
@@ -24,39 +24,87 @@ func TestStorageClusterHelmTemplate(t *testing.T) {
 		helmOption     *helm.Options
 		resultFileName string
 	}{
+		// {
```

Comment:
why is following tests commented?

Reply:
Removed
commented it by mistake while running tests.